api/xds_protocol.rst

```diff
@@ -135,6 +135,11 @@ versioning across resource types. When ADS is not used, even each
 resource of a given resource type may have a distinct version, since the
 Envoy API allows distinct EDS/RDS resources to point at different :ref:`ConfigSources <envoy_api_msg_core.ConfigSource>`.
 
+Only the first request on a stream is guaranteed to carry the node identifier.
+The subsequent discovery requests on the same stream carry an empty node
+identifier. This holds true regardless of the acceptance of the discovery
+responses on the same stream.
```

Comment:
I would also point out that the `Node`, if present more than once, should always be identical. It's sufficient to only check the first message for `Node` as a result.

Reply:
Done.